seatunnel-e2e/seatunnel-connector-v2-e2e/connector-http-e2e/src/test/java/org/apache/seatunnel/e2e/connector/http/HttpIT.java

```diff
@@ -176,7 +239,53 @@ public void testMultiTableHttp(TestContainer container)
             throws IOException, InterruptedException {
         Container.ExecResult execResult = container.executeJob("/fake_to_multitable.conf");
         Assertions.assertEquals(0, execResult.getExitCode());
-        Assertions.assertTrue(execResult.getStdout().contains(successCount));
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        String mockResponse =
+                mockServerClient.retrieveRecordedRequests(
+                        request().withPath("/example/httpMultiTableContentSink").withMethod("POST"),
+                        Format.JSON);
+        List<Record> recordResponse =
+                objectMapper.readValue(mockResponse, new TypeReference<List<Record>>() {});
+        recordResponse =
+                recordResponse.stream()
+                        .sorted(
+                                (r1, r2) ->
+                                        r1.getBody().getJson().getId()
+                                                - r2.getBody().getJson().getId())
+                        .collect(Collectors.toList());
+        Assertions.assertIterableEquals(records, recordResponse);
```

Comment:
```suggestion
        String mockResponse =
                mockServerClient.retrieveRecordedRequests(
                        request().withPath("/example/httpMultiTableContentSink").withMethod("POST"),
                        Format.JSON);
        Assertions.assertIterableEquals("{ ... }", mockResponse);
```
Why not check response string without any convert? This way more cleaier.

Reply:
Because the fields returned by mockresponse contain a lot of server information, which may not be the properties of our HTTP multi table relationship. So it is converted into a seatunnel row,
Do you think comparing JSON directly is enough?
![image](https://github.com/user-attachments/assets/75dff3e4-a3d8-4ad4-8141-85d400fbb249)

Reply:
Could you share the all value of mockResponse?

Reply:
[ {
  "body" : {
    "contentType" : "application/json",
    "type" : "JSON",
    "json" : {
      "id" : 2,
      "val_bool" : true,
      "val_int8" : 1,
      "val_int16" : 2,
      "val_int32" : 3,
      "val_int64" : 4,
      "val_float" : 4.3,
      "val_double" : 5.3,
      "val_decimal" : 6.3,
      "val_string" : "NEW",
      "val_unixtime_micros" : "2020-02-02T02:02:02"
    },
    "rawBytes" : "eyJpZCI6MiwidmFsX2Jvb2wiOnRydWUsInZhbF9pbnQ4IjoxLCJ2YWxfaW50MTYiOjIsInZhbF9pbnQzMiI6MywidmFsX2ludDY0Ijo0LCJ2YWxfZmxvYXQiOjQuMywidmFsX2RvdWJsZSI6NS4zLCJ2YWxfZGVjaW1hbCI6Ni4zLCJ2YWxfc3RyaW5nIjoiTkVXIiwidmFsX3VuaXh0aW1lX21pY3JvcyI6IjIwMjAtMDItMDJUMDI6MDI6MDIifQ=="
  },
  "headers" : {
    "content-length" : [ "193" ],
    "content-encoding" : [ ".*" ],
    "Connection" : [ "Keep-Alive" ],
    "User-Agent" : [ "Apache-HttpClient/4.5.13 (Java/1.8.0_342)" ],
    "Host" : [ "mockserver:1080" ],
    "Accept-Encoding" : [ "gzip,deflate" ],
    "token" : [ "9e32e859ef044462a257e1fc76730066" ],
    "Content-Type" : [ "application/json" ]
  },
  "keepAlive" : true,
  "method" : "POST",
  "path" : "/example/httpMultiTableContentSink",
  "secure" : false
}, {
  "body" : {
    "contentType" : "application/json",
    "type" : "JSON",
    "json" : {
      "id" : 1,
      "val_bool" : true,
      "val_int8" : 1,
      "val_int16" : 2,
      "val_int32" : 3,
      "val_int64" : 4,
      "val_float" : 4.3,
      "val_double" : 5.3,
      "val_decimal" : 6.3,
      "val_string" : "NEW",
      "val_unixtime_micros" : "2020-02-02T02:02:02"
    },
    "rawBytes" : "eyJpZCI6MSwidmFsX2Jvb2wiOnRydWUsInZhbF9pbnQ4IjoxLCJ2YWxfaW50MTYiOjIsInZhbF9pbnQzMiI6MywidmFsX2ludDY0Ijo0LCJ2YWxfZmxvYXQiOjQuMywidmFsX2RvdWJsZSI6NS4zLCJ2YWxfZGVjaW1hbCI6Ni4zLCJ2YWxfc3RyaW5nIjoiTkVXIiwidmFsX3VuaXh0aW1lX21pY3JvcyI6IjIwMjAtMDItMDJUMDI6MDI6MDIifQ=="
  },
  "headers" : {
    "content-length" : [ "193" ],
    "content-encoding" : [ ".*" ],
    "Connection" : [ "Keep-Alive" ],
    "User-Agent" : [ "Apache-HttpClient/4.5.13 (Java/1.8.0_342)" ],
    "Host" : [ "mockserver:1080" ],
    "Accept-Encoding" : [ "gzip,deflate" ],
    "token" : [ "9e32e859ef044462a257e1fc76730066" ],
    "Content-Type" : [ "application/json" ]
  },
  "keepAlive" : true,
  "method" : "POST",
  "path" : "/example/httpMultiTableContentSink",
  "secure" : false
} ]

Reply:
Ok. I get it.